test/scripts/auctionCancel.sh

```diff
@@ -71,7 +73,7 @@ export GENESIS_HASH="$(${NODEBINDIR}/goal node status -d ${ALGOTESTDIR} | grep '
 export TXN_FEE=1000
 
 {
-  ${NODEBINDIR}/auctionmaster -dir ${AUCTIONMASTERTESTDIR} -txround ${FINAL_ROUND} -notesfee ${TXN_FEE} -payfee ${TXN_FEE} -currentversion ${CURRENT_VERSION} -genhash ${GENESIS_HASH} -cancel true &> ${AUCTIONMASTERTESTDIR}/auction_master.log
```

Comment:
Does this break the regular integration target?

Reply:
Updated to support integration testing outside of mule environment.